app/controllers/RaceController.java

```diff
@@ -60,38 +56,31 @@ public RaceController(RaceRepository raceRepository) {
         });
     }
 
+    /*
     @BodyParser.Of(BodyParser.Json.class)
     public CompletionStage<Result> setRace() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)